celements-model/src/main/java/com/celements/store/CelHibernateStore.java

```diff
@@ -287,6 +349,14 @@ public ModelUtils getModelUtils() {
     return modelUtils;
   }
 
+  public String serialize(DocumentReference docRef, ReferenceSerializationMode mode) {

```

Comment:
It's a helper method which is also used in the store parts, e.g. `java.com.celements.store.part.CelHibernateStoreDocumentPart`. Since these parts are in a subpackage they cannot be accessed if they aren't public.
These public methods aren't part of the store interface so they won't be visible for store clients though (unless casted down explicitely).